Traceback (most recent call last):
  File "E:\JYZ_projects_python\J230724_NLP_lab640\J240601_TPLinker\TPlinker-ycy-main\TPLinker\tplinker\train.py", line 109, in <module>
    train_data = json.load(open(train_data_path, "r", encoding = "utf-8"))
FileNotFoundError: [Errno 2] No such file or directory: '../data4bert\\baidu_relation\\valid_data.json'
{'train_data': 'train_data.json', 'valid_data': 'valid_data.json', 'train_test_data': 'v_test.json', 'rel2id': 'rel2id.json', 'logger': 'wandb', 'f1_2_save': 0, 'fr_scratch': True, 'note': 'start from scratch', 'model_state_dict_path': '', 'hyper_parameters': {'shaking_type': 'cat', 'inner_enc_type': 'lstm', 'dist_emb_size': -1, 'ent_add_dist': False, 'rel_add_dist': False, 'match_pattern': 'only_head_text', 'lr': 5e-05, 'batch_size': 25, 'epochs': 50, 'seed': 2333, 'log_interval': 10, 'max_seq_len': 100, 'sliding_len': 20, 'loss_weight_recover_steps': 6000, 'scheduler': 'CAWR', 'T_mult': 1, 'rewarm_epoch_num': 2}, 'exp_name': 'baidu_relation', 'encoder': 'BERT', 'run_name': 'TP1+cat+BERT', 'data_home': '../data4bert', 'bert_path': '/home/yuanchaoyi/BeiKe/QA_match/roberta_base'}